index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
```

Comment:
- [ ] You should have mentioned that the body is not divided into semantically meaningful section for SEO and accessibility reasons. For example, `header, section, main, footer, etc`